app.js

```diff
@@ -144,6 +147,8 @@ function get_widgets(SETTINGS) {
         attribute: 'categories',
         operator: 'and',
         showMore: true,
+        searchable: true,
+        searchableIsAlwaysActive: false
```

Comment:
This is also nice! So you can search on categories, would love to have this too!